SparkPost/2web2ui
# SA-1514: Fixed date precision ranges

### What Changed
 - 15 minute precision restricted up to 48 hours
 - 5 minute precision restricted up to 24 hours
 - 1 minute precision restricted up to 24 hours

### How To Test
 - Go to report builder (analytics report)
 - Verify that precisions work correctly
 - Verify that precisions are selected correctly when relative date ranges in the date picker

### To Do
